plugins/techdocs/package.json

```diff
@@ -63,85 +63,8 @@
     "msw": "^0.21.2"
   },
   "files": [
-    "dist"
+    "dist",
+    "config.d.ts"
   ],
-  "configSchema": {
-    "$schema": "https://backstage.io/schema/config-v1",
-    "title": "@backstage/techdocs",
-    "type": "object",
-    "properties": {
-      "techdocs": {
-        "type": "object",
-        "properties": {
-          "requestUrl": {
-            "type": "string",
-            "visibility": "frontend"
-          },
-          "storageUrl": {
-            "type": "string",
-            "visibility": "backend"
-          },
-          "generators": {
-            "type": "object",
-            "properties": {
-              "techdocs": {
-                "type": "string",
-                "visibility": "backend"
-              }
-            }
-          },
-          "builder": {
-            "type": "string",
-            "visibility": "frontend"
-          },
-          "publisher": {
-            "oneOf": [
```

Comment:
Is it possible to do this `oneOf` in the file format of `config.d.ts`?

Reply:
Yep, using a union type